diff --git a/Makefile b/Makefile
index 039af00..c911e8e 100755
--- a/Makefile
+++ b/Makefile
@@ -8,7 +8,7 @@
 #TUNE = native
 
 CC = gcc
-CFLAGS = -O2 -Wall -I/usr/include -I./include
+CFLAGS = -O2 -Wall -I/usr/include -I./include -g
 OFILES = main.o comm.o
 
 INSTALLSITEMAN1DIR = ${mandir}/man1
@@ -136,8 +136,8 @@ clean:
 	rm -f $(TARGET) *.o
 
 install: aurora
-	install -m 4711 $(TARGET) $(bindir)
-	install -m 0644 ${DISTMAN1DIR}/$(TARGET).1 ${INSTALLSITEMAN1DIR}
+	install -D -m 4711 $(TARGET) $(DESTDIR)$(bindir)/$(TARGET)
+	install -D -m 0644 ${DISTMAN1DIR}/$(TARGET).1 $(DESTDIR)${INSTALLSITEMAN1DIR}/$(TARGET).1
 
 uninstall:
 	rm -f $(bindir)/$(TARGET) ${INSTALLSITEMAN1DIR}/$(TARGET).1
